<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>211</width>
    <height>346</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MedCar</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pbStart">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>310</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>START</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="tmOchtend">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>120</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="tmMiddag">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QTimeEdit" name="tmAvond">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QSpinBox" name="spBuzzer">
    <property name="geometry">
     <rect>
      <x>70</x>
      <y>260</y>
      <width>41</width>
      <height>21</height>
     </rect>
    </property>
    <property name="minimum">
     <number>1</number>
    </property>
    <property name="maximum">
     <number>60</number>
    </property>
    <property name="value">
     <number>5</number>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbOchtend">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>120</y>
      <width>95</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Ochtend</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbMiddag">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>160</y>
      <width>95</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Middag</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbAvond">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>200</y>
      <width>95</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Avond</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="cbTest">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>310</y>
      <width>61</width>
      <height>27</height>
     </rect>
    </property>
    <property name="text">
     <string>Test</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>240</y>
      <width>91</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Waarschuw</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>70</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tijden om medicijnen in te nemen:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>260</y>
      <width>51</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>om de</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>260</y>
      <width>61</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>minuten.</string>
    </property>
   </widget>
   <widget class="QTimeEdit" name="tmDraai">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>30</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>10</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Tijd om de carrousel te draaien:</string>
    </property>
    <property name="wordWrap">
     <bool>true</bool>
    </property>
   </widget>
   <zorder>label_5</zorder>
   <zorder>pbStart</zorder>
   <zorder>tmOchtend</zorder>
   <zorder>tmMiddag</zorder>
   <zorder>tmAvond</zorder>
   <zorder>spBuzzer</zorder>
   <zorder>cbOchtend</zorder>
   <zorder>cbMiddag</zorder>
   <zorder>cbAvond</zorder>
   <zorder>cbTest</zorder>
   <zorder>label</zorder>
   <zorder>label_2</zorder>
   <zorder>label_3</zorder>
   <zorder>label_4</zorder>
   <zorder>tmDraai</zorder>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
